# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: bc5b6dcb7057
# Full Hash: bc5b6dcb7057cf610efe29e14b65e9c3aa34d1c3
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-10-12 15:27:19
# Description:
#   Bug 1669162 - Update mapped_hyph to 0.4.2 so that .dic parse errors are non-fatal. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D92435
# ==============================================================================

diff -r a083e04138aa -r bc5b6dcb7057 .cargo/config.in
--- a/.cargo/config.in	Mon Oct 12 09:52:22 2020 +0000
+++ b/.cargo/config.in	Mon Oct 12 10:23:27 2020 +0000
@@ -40,7 +40,7 @@
 [source."https://github.com/jfkthame/mapped_hyph.git"]
 git = "https://github.com/jfkthame/mapped_hyph.git"
 replace-with = "vendored-sources"
-tag = "v0.4.0"
+tag = "v0.4.2"
 
 [source."https://github.com/hsivonen/packed_simd"]
 git = "https://github.com/hsivonen/packed_simd"